thirdparty/build.sh

```diff
@@ -5,13 +5,14 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-
-# build CLU
-mkdir -p CLU
-cd CLU
-cmake -DCMAKE_CXX_COMPILER=g++ -DCMAKE_C_COMPILER=gcc ../CLU.git
+# noma-ocl
+# Standalone/Library Build
+cd ocl
+mkdir -p build
+cd build
+cmake -DNOMA_OCL_STANDALONE=TRUE
```

Comment:
`..` at the end needed, also the build type `Release` should be set.

Reply:
Done with 1b6a3fa8e4ecbdd8c781c3b0e5ee0673a482e07b.